debezium-connector-mongodb/src/test/java/io/debezium/connector/mongodb/FiltersTest.java

```diff
@@ -37,80 +37,102 @@ public void beforeEach() {
         assertThat(filters.databaseFilter().test("mongo2")).isTrue();
     }
 
     @Test
-    public void shouldExcludeDatabaseCoveredByLiteralInBlacklist() {
+    public void shouldExcludeDatabaseCoveredByLiteralInExcludeList() {
         filters = build.excludeDatabases("db1").createFilters();
         assertThat(filters.databaseFilter().test("db1")).isFalse();
+
+        filters = build.useLiteralFilters().createFilters();
+        assertThat(filters.databaseFilter().test("db1")).isFalse();
     }
 
     @Test
-    public void shouldExcludeDatabaseCoveredByMultipleLiteralsInBlacklist() {
+    public void shouldExcludeDatabaseCoveredByMultipleLiteralsInExcludeList() {
         filters = build.excludeDatabases("db1,db2").createFilters();
         assertThat(filters.databaseFilter().test("db1")).isFalse();
         assertThat(filters.databaseFilter().test("db2")).isFalse();
+
+        filters = build.useLiteralFilters().createFilters();
+        assertThat(filters.databaseFilter().test("db1")).isFalse();
+        assertThat(filters.databaseFilter().test("db2")).isFalse();
     }
 
     @Test
-    public void shouldNotExcludeDatabaseNotCoveredByLiteralInBlacklist() {
+    public void shouldNotExcludeDatabaseNotCoveredByLiteralInExcludeList() {
         filters = build.excludeDatabases("db1").createFilters();
         assertThat(filters.databaseFilter().test("db2")).isTrue();
+
+        filters = build.useLiteralFilters().createFilters();
+        assertThat(filters.databaseFilter().test("db2")).isTrue();
     }
 
     @Test
-    public void shouldExcludeDatabaseCoveredByWildcardInBlacklist() {
+    public void shouldExcludeDatabaseCoveredByWildcardInExcludeList() {
         filters = build.excludeDatabases("db.*").createFilters();
         assertThat(filters.databaseFilter().test("db1")).isFalse();
     }
 
     @Test
-    public void shouldExcludeDatabaseCoveredByMultipleWildcardsInBlacklist() {
+    public void shouldExcludeDatabaseCoveredByMultipleWildcardsInExcludeList() {
         filters = build.excludeDatabases("db.*,mongo.*").createFilters();
         assertThat(filters.databaseFilter().test("db1")).isFalse();
         assertThat(filters.databaseFilter().test("mongo2")).isFalse();
     }
 
     @Test
-    public void shouldIncludeCollectionCoveredByLiteralWithPeriodAsWildcardInWhitelistAndNoBlacklist() {
+    public void shouldIncludeCollectionCoveredByLiteralWithPeriodAsWildcardInIncludeListAndNoExcludeList() {
         filters = build.includeCollections("db1.coll[.]?ection[x]?A,db1[.](.*)B").createFilters();
         assertCollectionIncluded("db1xcoll.ectionA"); // first '.' is an unescaped wildcard in regex
         assertCollectionIncluded("db1.collectionA");
     }
 
     @Test
-    public void shouldIncludeCollectionCoveredByLiteralInWhitelistAndNoBlacklist() {
+    public void shouldIncludeCollectionCoveredByLiteralInIncludeListAndNoExcludeList() {
         filters = build.includeCollections("db1.collectionA").createFilters();
```

Comment:
Do we have any tests for whitespace with commas separated values for literals? I didn't see it.

Reply:
We do now. 

@jpechane as per @btiernay request the "literal" matching is now trimming away whitespaces. Meaning `dbA.col1,dbB.col1` is the same as ` dbA.col1 ,  dbB.col1`, is this OK?